app/Controllers/Http/RecipeController.ts

```diff
@@ -121,6 +123,7 @@ export default class RecipesController {
     await Recipe.create({
       name: data.name,
       recipeId: data.id,
+      // @ts-expect-error
```

Comment:
I usually take the TS error message and put it there like `// @ts-expect-error foo` to make it more self-explanatory what the problem is.